app/views/vacancies/_show.html+phone.haml

```diff
@@ -30,56 +30,51 @@
-
+    %dl.app-check-your-answers.app-check-your-answers--short
+      .app-check-your-answers__contents
+        %dt.app-check-your-answers__question= t('jobs.salary')
+        %dd.app-check-your-answers__answer= @vacancy.salary_range
+      .app-check-your-answers__contents
+        %dt.app-check-your-answers__question= t('jobs.working_pattern')
+        %dd.app-check-your-answers__answer= @vacancy.working_pattern
+      - if @vacancy.part_time? && @vacancy.weekly_hours
+        .app-check-your-answers__contents
+          %dt.app-check-your-answers__question= t('jobs.weekly_hours')
+          %dd.app-check-your-answers__answer= @vacancy.weekly_hours
+      - if @vacancy.flexible_working?
+        .app-check-your-answers__contents
+          %dt.app-check-your-answers__question= t('jobs.flexible_working')
+          %dd.app-check-your-answers__answer= @vacancy.flexible_working
+      - if @vacancy.starts_on.present?
+        .app-check-your-answers__contents
+          %dt.app-check-your-answers__question= t('jobs.starts_on')
+          %dd.app-check-your-answers__answer= format_date(@vacancy.starts_on)
+      - if @vacancy.ends_on.present?
+        .app-check-your-answers__contents
+          %dt.app-check-your-answers__question= t('jobs.ends_on')
+          %dd.app-check-your-answers__answer= format_date(@vacancy.ends_on)
+      .app-check-your-answers__contents
+        %dt.app-check-your-answers__question= t('jobs.publish_on')
+        %dd.app-check-your-answers__answer= format_date(@vacancy.publish_on)
+      .app-check-your-answers__contents
+        %dt.app-check-your-answers__question= t('jobs.expires_on')
+        %dd.app-check-your-answers__answer= format_date(@vacancy.expires_on)
+      - if @vacancy.main_subject.present?
+        .app-check-your-answers__contents
+          %dt.app-check-your-answers__question= t('jobs.main_subject')
+          %dd.app-check-your-answers__answer= @vacancy.main_subject
+      - if @vacancy.pay_scale_range.present?
+        .app-check-your-answers__contents
+          %dt.app-check-your-answers__question= t('jobs.pay_scale_range')
+          %dd.app-check-your-answers__answer= @vacancy.pay_scale_range
+      - if @vacancy.leadership.present?
+        .app-check-your-answers__contents
+          %dt.app-check-your-answers__question= t('jobs.leadership_level')
+          %dd.app-check-your-answers__answer= @vacancy.leadership.title
+      - if @vacancy.contact_email.present?
+        .app-check-your-answers__contents
+          %dt.app-check-your-answers__question= t('jobs.contact_email')
+          %dd.app-check-your-answers__answer.wordwrap= mail_to @vacancy.contact_email, @vacancy.contact_email, subject: t('jobs.contact_email_subject', job: @vacancy.job_title), body: t('jobs.contact_email_body', url: url_for(only_path: false))
```

Comment:
The blue on this link appears to be the unstyled one I think. 
![screen shot 2018-10-08 at 16 51 25](https://user-images.githubusercontent.com/912473/46619624-7823cb00-cb1a-11e8-96e3-13f44c2655cf.png)